tests/sentry/api/endpoints/test_project_rules_configuration.py

```diff
@@ -93,11 +93,4 @@ def test_available_actions(self):
 
         action_ids = [action["id"] for action in response.data["actions"]]
         assert EMAIL_ACTION in action_ids
-        assert JIRA_ACTION not in action_ids
-
-    def test_ticket_rules_in_available_actions(self):
-        with self.feature("organizations:integrations-ticket-rules"):
```

Comment:
@mgaeta We should make sure that when `organizations:integrations-ticket-rules` is `False` that we don't render the Jira action.

Reply:
I added the test back and updated the function names.